cypress/integration/tests/developer/applicationinventory/analysis/upload_binary_analysis.test.ts

```diff
@@ -40,8 +40,8 @@ describe("Upload Binary Analysis", { tags: "@tier1" }, () => {
         deleteApplicationTableRows();
 
         //Disable all proxy settings
-        let proxy = new Proxy(data.getRandomProxyData());
-        proxy.disableProxy();
+        // let proxy = new Proxy(data.getRandomProxyData());
+        // proxy.disableProxy();
```

Comment:
Why is this commented ? 
If not needed , please remove .

Reply:
Fixed